src/kayenta/stages/kayentaStage/kayentaStageExecutionDetails.controller.ts

```diff
@@ -52,6 +55,28 @@ class KayentaStageExecutionDetailsController {
     // The kayentaStageTransformer pushes related 'runCanary' and 'wait' stages
     // into the 'kayentaCanary' tasks list.
     this.canaryRuns = this.$scope.stage.tasks.filter((t: IExecutionStage) => t.type === RUN_CANARY);
+    this.canaryRuns.forEach((run: any) => {
+      const { canaryConfigId, canaryPipelineExecutionId } = run.context;
+      const metricsAccountName = get(run, ['context', 'metricsAccountName'], '');
+      const canaryRunPromise = getCanaryRun(canaryConfigId, canaryPipelineExecutionId);
+      const accountPromise = getKayentaAccount(metricsAccountName);
+      Promise.all([canaryRunPromise, accountPromise]).then(([runDetails, account]) => {
+        if (account && account.type === 'datadog') {
```

Comment:
Could you please leave a link to the Github discussion about why Datadog is omitted here?

Reply:
👍 done